Android/app/src/main/java/io/realm/scanner/MainActivity.java

```diff
@@ -373,9 +374,8 @@ public void onClick(DialogInterface dialogInterface, int i) {
 
     private File createImageFile() throws IOException {
         String timeStamp = new SimpleDateFormat("yyyyMMdd_HHmmss").format(new Date());
-        String imageFileName = "JPEG_" + timeStamp + "_";
         File storageDir = getExternalFilesDir(Environment.DIRECTORY_PICTURES);
-        File image = File.createTempFile(imageFileName, ".jpg", storageDir);
+        File image = new File(storageDir, "JPEG_" + timeStamp + ".jpg");
```

Comment:
I don't quite understand this. So the shared folder is `Pictures/` on the internal storage, but the images are still created in external `DIRECTORY_PICTURES`. Why do we need the `FileProvider` here?